lib/prepare_release.js

```diff
@@ -204,6 +210,33 @@ class ReleasePreparation {
     return this.config.username;
   }
 
+  warnForMissing() {
+    const { cli, upstream, username } = this;
+
+    const missing  = !username || !upstream;
```

Comment:
```suggestion
    const missing = !username || !upstream;
```
should fix the lint error.